Helm/opensearch-dashboards/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "opensearch-dashboards.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "opensearch-dashboards.standard" . | indent 4 }}
+rules:
+- apiGroups: ['extensions']
+  resources: ['podsecuritypolicies']
```

Comment:
Do we need this file to be written if PSP is disabled? It's only controller PSP related usage. 

Also, we don't even have a PSP policy in this chart so the role/rolebinding seems redundant.

Reply:
Good catch. As we don't have PSP for now, we can eliminate either of them.

Reply:
There is one more way to look at it from the future prospective, as this is not breaking and does not impact , we can leave as it is , in case we add a PSP , this can be utilised.

Reply:
"in case" - add it when it is actually used.
PSPs are deprecated anyways: https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/

Reply:
Makes sense , I will remove it for now.